README.md

```diff
@@ -4,33 +4,36 @@
 [![Build Status](https://travis-ci.org/php-parallel-lint/PHP-Parallel-Lint.svg?branch=master)](https://travis-ci.org/php-parallel-lint/PHP-Parallel-Lint)
 [![License](https://poser.pugx.org/php-parallel-lint/php-parallel-lint/license.svg)](https://packagist.org/packages/php-parallel-lint/php-parallel-lint)
 
-This tool checks syntax of PHP files faster than serial check with a fancier output.
+This application checks syntax of PHP files in parallel.
+It can output in plain text, colored text, json and checksyntax formats.
+Additionally `git blame` can be used to show commits that introduced the breakage.
```

Comment:
```suggestion
Additionally `blame` can be used to show commits that introduced the breakage.
```

`git blame` suggests that you'd have to run `git blame` after running this tooling, while it is actually build in via the `--blame` command-line option.